README.md

```diff
@@ -41,7 +41,7 @@ The following stable releases are supported. The development branch usually work
 ## Test status of cloud providers
 
 * [Betacloud](https://www.betacloud.de): Works
-* [Citycloud](https://www.citycloud.com): Works (need to change disk names from sdX to vdX, pass ``drives_vdx: true`` in environment)
+* [Citycloud](https://www.citycloud.com): Works (need to change disk names from sdX to vdX, pass ``drives_vdx: true`` in environment). You can use ``environment-CC.yml`` to get a set of working parameters.
```

Comment:
i'm not a big fan of unnecessary shortcuts. Can we use environment-citycloud.yml?

Reply:
Fair enough, I have renamed it now.